### PR TITLE
Fix angular tests 

### PR DESCRIPTION
**Pull request by holachuy**
Fix angular tests so they all pass including testing component creation and fixing form template parsing errors. Issues  #5 #22

### CODE REVIEW

File: portability-core/src/test/java/org/dataportabilityproject/job/JobDaoTest.java [hunk 1]
```diff
@@ -34,7 +35,7 @@
 
   @Before
   public void setUp() {
-    jobDao = new JobDao(new LocalCloudFactory().getPersistentKeyValueStore());
```
Comment (rtannenbaum) [inline portability-core/src/test/java/org/dataportabilityproject/job/JobDaoTest.java]: should this be in a separate PR?

Comment (holachuy) [reply portability-core/src/test/java/org/dataportabilityproject/job/JobDaoTest.java]: Done! Good catch.
